### ISSUE TITLE
Add another Hetzner node to Harvester

### ISSUE THREAD

**Issue by meysholdt** [gitpod-io/gitpod#9000]
We have one more (currently unused) dedicated server on Hetzner. We should add it to the harvester cluster. 

proposed solution:
1. install CentOS with same network config as on other Harvester nodes.
2. install one (instead of two) Havester nodes on it and allocate ~95% of the resources. We don't need to have two smaller nodes here, because we already have enough nodes to run Harvester in HA mode.

**Comment by meysholdt** [comment]
Hetzner already did install CentOS on server `hetzner2`. Password is in 1password. 

Proposed solution:
a) install Harvester 1.0.0 on this node so that it can be upgraded together with https://github.com/gitpod-io/gitpod/issues/8999.
b) fix https://github.com/gitpod-io/gitpod/issues/8999 first and then install Harvester 1.0.1 on this node.

**Comment by wulfthimm** [comment]
I updated the system, enabled the firewall, installed the virtualization system and created a VLAN with a bridge device.

**Comment by wulfthimm** [comment]
I am adding the `blocked` label until we have solved https://github.com/gitpod-io/gitpod/issues/8999 and can directly install Harvester 1.0.1 on hetzner2.

**Comment by meysholdt** [comment]
I've created a `centvester5` node, but it can't connect to the cluster because there is trouble with the LetsEncrypt certificate:
<img width="1049" alt="image" src="https://user-images.githubusercontent.com/239422/164505780-d30d64f8-7ef1-4e8a-a3c6-a53108e1d7e2.png">

I suspect the LetsEncrypt CA is not installed on the harvester Node.

I also tried to provide the IP (instead of the domain) of the Harvester management node. This also got rejected because the IP is not listed in the Cert.
